distro/tests/add_test.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: MPL-2.0
+
+# Usage: ./add_test.sh <test_name>
```

Comment:
As a potential user of this test framework, I hate to read or learn any new script or tool. Why not provide a template directory so that the user can simply `cp -r template <my_test>`? This removes the magic script and everything will be straightforward.